src/components/project-overview/project-overview.scss

```diff
@@ -27,21 +27,15 @@
     gap: 2rem;
     max-width: 100%;
 
-    @media only screen and (max-width:640px) {
+    @media only screen and (max-width: 640px) {
       flex-direction: column;
     }
-
   }
 
   &__card {
-    width: 384px;
-    height: 594px;
```

Comment:
These were causing this to happen:

![image](https://github.com/CodeForBc/codeforbc-web/assets/16601729/79203d0e-0921-442c-a292-856e6b333226)